### DISCUSSION TITLE
Latest release enables support for property grouping in profiles

### DISCUSSION THREAD

**Discussion by marcolarosa** [Arkisto-Platform/describo-online#49] [Announcements]
You will have noticed that when you provide a profile to describo all of the properties are displayed in one big list. As of today's work you can now define a tabbed layout per entity type and within that, order which properties are shown on each tab and in what order.

Read the docs at https://github.com/Arkisto-Platform/describo-online/wiki/Writing-a-domain-specific-profile#layouts

**Comment by juriroemer** [comment]
This seems like a very interesting and useful feature - I think it can greatly improve end user usability, if implemented correctly by profile creators!

I have a question though, looking and the docs: 
> If the profile defines other properties that are not listed in the layout they will be added to a final tab.

Is that the `other` tab shown in the screenshot? That's what I would assume from the name, but on the other hand, the `other` tab isn't the final one, but `Linked to` is. Or is `Linked to` also implicitly generated?

From looking at this, I'm assuming the profile `json` shown does not directly correlate to the screenshot. If that is the case, could you maybe use the `json` that was actually used for the screenshot as the example? Or if in fact that `json` was used for the screenshot, could you explain in the docs where `Core Metadata`, `other` and `Linked To` come from?

**Comment by marcolarosa** [comment]
Hi @juriroemer 


All great comments. I'm actually not happy with the label `other` so I'm changing it to `...` for now. If you have some better ideas please let us know.

As for `Linked to` - this tab shows what the entity is linked to but it can go on the first tab with @id, @type and name which I've done. That way, one less tab and the left over entities (destined for ...) will be on the last tab.

And finally, do you have an alternative name for the first tab? @ptsefton suggests `about` which I like as well and am changing now but if you have another name you think would work mark it here. Core Metadata was my choice to get started but it's not a good name because it's likely more useful as the first tab with actual fields to fill in.

**Reply by juriroemer** [reply]
Hey @marcolarosa, as we are still using describo-online until we have ported the front-end to vue3, a quick question: is it possible to hide the `About` tab? I've tried [Hide](https://github.com/Arkisto-Platform/describo-online/wiki/Writing-a-domain-specific-profile#hide-layout-properties) with no luck.

**Reply by marcolarosa** [reply]
Hi @juriroemer No it's not. This is also true for the profiles that the new component understands but in that case, you can at least add other properties to that about tab: https://github.com/describo/profiles/blob/master/profile-docs/hide-and-layouts.md#adding-properties-to-the-about-tab

In terms of this product, to my knowledge it is unmaintained. Any effort I can spare goes to the new component (some work overlaps with my day job but most of that development is now done on a best effort basis when i can give it some time). I suggest you work towards using the new component in your environment.

**Reply by juriroemer** [reply]
> In terms of this product, to my knowledge it is unmaintained.

Yes, yes, was just wondering if there already was a way that I had missed, no expectations. Thanks!
